service/model_manager/model_installer.py

```diff
@@ -9,6 +9,8 @@
 import threading
 import server
 import time
+from .missing_models import find_missing_models # type: ignore noqa
```

Comment:
if there is a better way to import the entire missing_models, please let me know. Im not very familiar how python import works. I don't like the unnecessary find_missing_models func import either

Reply:
i think we can store in both actually, they don't conflict